admin/tabs/userData/roles/RoleModel.ts

```diff
@@ -136,6 +143,19 @@ export class RoleModel extends HoistModel {
         return true;
     }
 
+    async changeCategoryAsync(roles: HoistRole[], category: string): Promise<void> {
+        if (this.readonly) return;
+        const roleNames: string[] = map(roles, it => it.name);
+        await XH.fetchService.postJson({
+            url: 'roleAdmin/bulkCategoryUpdate',
+            body: {
+                roleNames,
```

Comment:
this argument can just be "roles" obvious that it has to be the unique id, e.g. name